### ISSUE TITLE
Wix Importer Error: "It looks like we're unable to reach that website address at this time"

### ISSUE THREAD

**Issue by eduardozulian** [Automattic/wp-calypso#45586]
#### Steps to reproduce

1. Starting at URL: https://wordpress.com/import/
2. Select a site to import into (if applicable)
3. Click Wix
4. Enter a Wix import URL like https://velda7.wixsite.com/velda7 (thanks @supernovia)

#### What I expected

The import to start.

#### What happened instead

An error:

```
It looks like we're unable to reach that website address at this time. (ID: 7241f121-2f35-44db-a387-3e3a1b82b223)
```

If you hit the `Try again` button, another error ID is generated

#### Screenshot / Video

<img width="754" alt="Captura de Tela 2020-09-11 às 12 24 47" src="https://user-images.githubusercontent.com/1136160/92944057-e28dc600-f429-11ea-96bd-fafecbfd51bb.png">

#### Context / Source

#user-report in 3284417-zen

We thought this was related to #44503 but given it's a different error message it's best to open a new issue.

**Comment by nagpai** [comment]
#user-report in  3302747-zen  and 3314347-zen

**Comment by jackiejade** [comment]
another #user-report in 3315862-zen

**Comment by automattic-ian** [comment]
Additional user report in  #3281790-zen

**Comment by aisajib** [comment]
Another user reported this in #24202809-zen

**Comment by yoavf** [comment]
Seems pretty random - after about 10 attempts (bypassing the caching of the site parser) I was able to import the site mentioned in the OP - https://velda7.wixsite.com/velda7 

We should look into automating the retries.

**Comment by jordesign** [comment]
Another user report in 3346900-zd

**Comment by yoavf** [comment]
> Another user report in 3346900-zd

I was able to complete that import as well.

**Comment by supernovia** [comment]
Noting I must have tried a dozen or more times without success.

**Comment by yoavf** [comment]
> Noting I must have tried a dozen or more times without success.

Yep, it'd be pretty hard to get to success without turning off the caching in the code, so I really don't expect it to work at this moment.

**Comment by pauljacobson** [comment]
> Yep, it'd be pretty hard to get to success without turning off the caching in the code, so I really don't expect it to work at this moment.

@yoavf Does this mean that we don't have a solution, and it really just comes down to trying repeatedly until it works?

**Comment by yoavf** [comment]
Update: workaround in DD50844-code

**Comment by formosattic** [comment]
Issue still happening. 24844791-hc

**Comment by Agepeuve** [comment]
We got another report in 3399795-zd

**Comment by yoavf** [comment]
@formosattic and @Agepeuve - we've just posted our internal workaround. 
Please see p7DVsv-9Cf-p2 - and let me know how that works.

**Comment by kylemcph** [comment]
Another report in 25404742-hc. The workaround mentioned above worked perfectly. 🙇‍♂️

**Comment by supernovia** [comment]
https://wordpress.com/forums/topic/import-from-wix-2/ Another case here. Waiting on a bit more information before trying the fix.

**Comment by katiebethbrown** [comment]
Another report in 26008564-hc. I ran the importer for the user and was able to get it to successfully complete after a few tries.

**Comment by Neibaf** [comment]
I tried several times to import via desktop app or via wordpress.com and I still have the same error:

`It looks like we're unable to reach that website address at this time. (ID: 0b058c82-0d07-4cb1-911e-72e05aaade49)`

**Comment by yoavf** [comment]
@Neibaf please [contact our support team](https://wordpress.com/help/contact/) and they'll be happy to assist you with this.

**Comment by supernovia** [comment]
Another case here: https://wordpress.com/forums/topic/problem-transferring-site-from-wix/  -- we'll update if we can get it imported

**Comment by supernovia** [comment]
And another case here: https://wordpress.com/forums/topic/wix-import/

**Comment by yoavf** [comment]
@supernovia I should have closed this issue as it was resolved on our end a few months ago. But two recent reports tell me that this might be a new problem. Please share those URLs with us when the customers do - we'll take a look.

**Comment by macmanx2** [comment]
Hi there @yoavf, for https://wordpress.com/forums/topic/problem-transferring-site-from-wix/ that's https://ebonyro1.wixsite.com/ebonynikita to PcQxzG-5-p2

I tried repeating the import 10 times per our internal workaround, but no luck. :(

**Comment by yoavf** [comment]
> Hi there @yoavf, for https://wordpress.com/forums/topic/problem-transferring-site-from-wix/ that's https://ebonyro1.wixsite.com/ebonynikita to [PcQxzG-5-p2](https://href.li/?PcQxzG-5-p2)
> 
> I tried repeating the import 10 times per our internal workaround, but no luck. :(

Thanks - I can confirm the issue with this site, and am looking at it.

**Comment by yoavf** [comment]
> Hi there @yoavf, for https://wordpress.com/forums/topic/problem-transferring-site-from-wix/ that's https://ebonyro1.wixsite.com/ebonynikita to [PcQxzG-5-p2](https://href.li/?PcQxzG-5-p2)
> 
> I tried repeating the import 10 times per our internal workaround, but no luck. :(

For this site, the issue was due to some posts requiring a subscription to view. 
The underlying problem was fixed, and the importer should work again as expected. I let the user know in the thread.

**Comment by csonnek** [comment]
@yoavf the user reported back: "Only the pages transferred but not all of the elements like the pictures or titles. Also the main homepage did not transfer over."

https://wordpress.com/forums/topic/problem-transferring-site-from-wix/#post-3653577

I'm not sure if those items are meant to be transferred to show exactly like their Wix site, so could you advise if this is the expected behavior or if there's still a problem here? Thanks!

**Comment by galakhyati** [comment]
@yoavf another report found here: 29389857-hc

Wix site they are importing from: https://www.boomelectrical.com


https://user-images.githubusercontent.com/7382505/116989403-50576c80-acef-11eb-9564-e655ed9871ac.mp4

**Comment by akirk** [comment]
For reference, the problem encountered in 29389857-hc was fixed.

**Comment by ImNotRice** [comment]
Same issue as everyone else here.

**Comment by csonnek** [comment]
@yoavf Another report in https://wordpress.com/forums/topic/having-trouble-importing-site-from-wix/

Error message: `It looks like we’re unable to reach that website address at this time. (ID: 9a97ea60-6c0a-4873-8976-038f392280bd)`

**Comment by supernovia** [comment]
Noting I tried importing for the user [in this thread](https://wordpress.com/forums/topic/having-trouble-importing-site-from-wix/) and received the "It looks like we’re unable to reach that website address at this time" error repeatedly.

**Comment by kosiew** [comment]
Tested and encountered the error a few times too.

This is already on a project board.
@vishnugopal can you assign have a look at this?

**Comment by donalirl** [comment]
Seen in #34843674-hc p7DVsv-9Cf-p2#comment-40058

**Comment by vishnugopal** [comment]
@donalirl Will re-triage this and get back, thanks for the report!

**Comment by KokkieH** [comment]
Another report in https://wordpress.com/forums/topic/importing-from-wix-3/?view=all

**Comment by vishnugopal** [comment]
This seems to be a case that the original import site was not Wix. We'll have better error messages here soon, see: p7DVsv-9Cf-p2#comment-40514

**Comment by vishnugopal** [comment]
Improve Error messages tracked here: https://github.com/Automattic/wp-calypso/issues/63122

**Comment by KokkieH** [comment]
Thanks, @vishnugopal. Still waiting for the user to confirm the site URL in the forum thread - they haven't replied to me yet.

**Comment by csonnek** [comment]
Another report in: https://wordpress.com/forums/topic/i-cant-import-my-wix-site/?view=all

Site: https://www.prosperlaw.com.au/

**Comment by macmanx2** [comment]
user from the above thread reported:

> there is no error, it just goes through to a white screen and then when i go back it says ‘there has been a problem importing your site’.

**Comment by fstath** [comment]
I got another report in https://wordpress.com/forums/topic/help-importing-from-wix/?view=all

I tried importing it on one of my test sites and got the same error.

This is the error message:
`It looks like we're unable to reach that website address at this time. (ID: ee996d19-659d-4b05-ab20-222acf51f4a5)
`

Wix site address: https://norriserin.wixsite.com/callofthevoidblog

**Comment by sudeepbaral** [comment]
Another reported issue here: 5232505-zd

Tested it on my test site and I am also seeing an error `It looks like we're unable to reach that website address at this time. (ID: b8bc9eb4-c555-4198-af52-3320afc79ca6)`

Wix site: `karlclare.com`

I have asked them for Wix site address.

**Comment by jp-imagines** [comment]
Another report in 5235478-zd-woothemes: `It looks like we are unable to reach that website address at this time. (ID: 57e5a057-6587-4899-8311-427debc52276)`

Wix site: `dealerosity.com` (asking for the free Wix address now)

**Comment by fstath** [comment]
I got another report in https://wordpress.com/forums/topic/trying-to-import-my-wix-site-to-wordpress/?view=all

I tried importing it on one of my test sites and got the same error.

This is the error message:
`It looks like we’re unable to reach that website address at this time. (ID: f7317077-5224-415a-b384-50f8dd96038d)`

Wix site address: https://momentsbymikhail.wixsite.com/home

**Comment by csabarakasz** [comment]
Another report for this: 5261536-zd-woothemes Same thing!

Tested it on my site, and the same issue. No known workarounds at this time.

Wix site: www.prosperlaw.com.au
Free WP.com site: http://law452655160.wordpress.com/

Update: I found this workaround that worked for me, but it is hard to implement: https://themeisle.com/blog/transfer-from-wix-to-wordpress/

**Comment by vishnugopal** [comment]
All of these are now working for me, starting from Tools > Import for a simple site:

<img width="614" alt="CleanShot 2022-06-03 at 13 04 30@2x" src="https://user-images.githubusercontent.com/533/171809215-59d1550d-a1d9-4190-86bc-ea22713791a2.png">

<img width="624" alt="CleanShot 2022-06-03 at 13 05 13@2x" src="https://user-images.githubusercontent.com/533/171809394-da6df4e7-2a3c-471d-b049-40423973c3ff.png">

<img width="618" alt="CleanShot 2022-06-03 at 13 05 46@2x" src="https://user-images.githubusercontent.com/533/171809479-dd727339-672e-4411-a37f-11feb1c7a048.png">

<img width="609" alt="CleanShot 2022-06-03 at 13 06 20@2x" src="https://user-images.githubusercontent.com/533/171809588-2e1b28c5-0c79-4594-880d-222f261dec2c.png">

<img width="631" alt="CleanShot 2022-06-03 at 13 06 52@2x" src="https://user-images.githubusercontent.com/533/171809679-b703ea1e-c6b0-4345-af46-f5955bffdfff.png">


Might have been a transient error, so please double check!

**Comment by csabarakasz** [comment]
Hi, thanks for testing @vishnugopal ! After testing the www.prosperlaw.com.au Wix site again, which you mentioned in your post, I noticed that it still gives an error message. [Original comment with this site](https://github.com/Automattic/wp-calypso/issues/45586#issuecomment-1143424041).

```
Another report for this: 5261536-zd-woothemes Same thing!

Tested it on my site, and the same issue. No known workarounds at this time.

Wix site: www.prosperlaw.com.au
Free WP.com site: http://law452655160.wordpress.com/

Update: I found this workaround that worked for me, but it is hard to implement: https://themeisle.com/blog/transfer-from-wix-to-wordpress/
```

I could see the page you inserted as well, but the error message comes after I press **Yes! Start import**. 

Hold tight, the video is 45 sec :)
![CleanShot 2022-06-05 at 16 08 16](https://user-images.githubusercontent.com/84539773/172054685-b126e734-b004-420f-b446-dfc62adafbfe.gif)

**Comment by edwinho89** [comment]
17543059-hc

Wix importer page displays a blank preview, and when attempting to "Start Import" we had this error message:

```
It looks like we're unable to reach that website address at this time. (ID: 249a1f7d-ee22-40d2-8cca-9c4128d8dd0d)
Try again or get help.
```

Wix site: https://www.shesospoiled.com/

Suggesting workaround from previous comment: https://themeisle.com/blog/transfer-from-wix-to-wordpress/

**Comment by fstath** [comment]
I got another report at https://wordpress.com/forums/topic/importing-site-from-wix/?view=all

Wix URL: https://www.highrockyhomes.com/

**Comment by jerrysarcastic** [comment]
User in https://wordpress.com/forums/topic/importing-content-from-wix

`It looks like we're unable to reach that website address at this time. (ID: f3be65d6-4b88-44a1-b829-86079c10bb3b)`

I suggested this workaround as others have noted (after shaking my fist at the sky in solidarity with the site owner) https://themeisle.com/blog/transfer-from-wix-to-wordpress/

**Comment by Jaykame** [comment]
35902153-hc

Wix URL: latskitchen.com

Gave user the article as workaround: https://themeisle.com/blog/transfer-from-wix-to-wordpress/

**Comment by jamiepalatnik** [comment]
Another report: 30392241-hc

- [ ] Follow-up ticket: 5332709-zd-woothemes

Shared article as a workaround: https://themeisle.com/blog/transfer-from-wix-to-wordpress/

**Comment by kavyagokul** [comment]
Another report in 35951571-hc

They were getting 
`It looks like we're unable to reach that website address at this time. (ID: 159d1f61-2b3d-4c8d-91cb-5bfeb534ea6d)`

Shared article as workaround: https://themeisle.com/blog/transfer-from-wix-to-wordpress/

**Comment by ktyfuller604** [comment]
Another report in 35957387-hc

Shared article as workaround: https://themeisle.com/blog/transfer-from-wix-to-wordpress/

**Comment by zaerl** [comment]
After an initial analysis, It seems that this bug is random. I tried all the URLs reported here and I can't reproduce them. I do indeed see that there has been some errors in the near past.

`It looks like we're unable to reach that website address at this time.`

This means that the server tried to load the website but something happened on the other side. There can be multiple possibilities. I will search these calls' history to see if we can find an URL that can reproduce this bug 100% of the time.

**Comment by jerrysarcastic** [comment]
Also reported in https://wordpress.com/forums/topic/import-error-from-transfer

Shared the same workaround that others here have suggested

**Comment by vishnugopal** [comment]
We've fixed a bunch of issues here. See: 
* 417-gh-Automattic/site-parser
* https://github.com/Automattic/wp-calypso/issues/63122 

Closing this! Reopen this if the issue recurs!

**Comment by zdenys** [comment]
I've got a new case in 5967320-zd-woothemes I get the same when I try importing the site on my test site:

![7Pi2A4.png](https://user-images.githubusercontent.com/39308239/219695757-6551d821-2f23-4e3b-92d6-fd3169baa3d5.png) 

cc: @Automattic/caribou 🙇 

The user made the site unavailable at Wix for now since they don't want Google to index it) but they are willing to make it public again for us for a short agreed-upon period (hours) if we need it for investigating.

Thus, we need to contact them in 5967320-zd-woothemes to ask and make the site public. Let's just coordinate on the best time to respect the user's request from chat 40759658-hc:

> I can do that but it is my understanding that we only have a few hours or so until it gets indexed. Can we coordinate the timing so I can publish it while your team works on it and then unpublish it as soon as they don't need it anymore?

**Comment by IoanaAMuresan** [comment]
Asked this last customer to make the site available, as mentioned in the note above. Asked them to confirm when it's done.

**Comment by sudeepbaral** [comment]
The user on the last ticket has made their site public: https://www.placelinc.com/

cc: @Automattic/caribou 

We can go ahead and investigate the issue on their site

**Comment by hacchism** [comment]
> The user on the last ticket has made their site public: https://www.placelinc.com/
> 
> cc: @Automattic/caribou
> 
> We can go ahead and investigate the issue on their site

Since I anticipated it would take more days for us to investigate this, I introduced [the workaround](https://themeisle.com/blog/transfer-from-wix-to-wordpress/) to the user but the user said they prefer that we keep investigating this.

cc: @Automattic/caribou

**Comment by rw-ye** [comment]
Another report here - 5969087-zd-woothemes

Wix URL: https://www.bookwormsandbiblios.com/

**Comment by aisajib** [comment]
Another report: https://wordpress.com/forums/topic/about-installing-from-your-wix-siteabout-migration/?view=all
Site: https://www.kanetuu.co.jp/

**Comment by ariel-maidana** [comment]
Please notify 5967320-zen once a fix is in place.

**Comment by jerrysarcastic** [comment]
Reported in https://wordpress.com/forums/topic/import-245/

**Comment by jerrysarcastic** [comment]
Issue reported in https://wordpress.com/forums/topic/unable-to-import-from-wix/

error message (according to site owner)  `It looks like we’re unable to reach that website address at this time. (ID: dd04766e-27ac-464d-be9b-62a397800ce0)`

Importing from https://agitacion.wixsite.com/home

**Comment by mgozdis** [comment]
Another report here: 6188235-zd-woothemes
Importing from https://www.butterfliesoforegon.com

They are getting a 403 on the preview. They then get random 504 responses and `It looks like we're unable to reach that website address at this time. ` when importing.

**Comment by geekinthegirl** [comment]
I've just come across a user in tickets with the same issue: 6299332-zen

Here's the error: `It looks like we're unable to reach that website address at this time. (ID: ea244707-afa5-4e95-974e-03da351cee82)
Try again or get help.`

I've suggested the workaround here for now: https://themeisle.com/blog/transfer-from-wix-to-wordpress/

**Comment by thesacredpath** [comment]
Another one here, but 500 error with 403 in the preview. Tried the import over 10 times and no joy. Follow up from chat below. Importing from: gethandi.com

6317650-zd-woothemes

**Comment by cuemarie** [comment]
Escalated with @Automattic/caribou here: p1687488633273859-slack-C01A60HCGUA

**Comment by filipanoscampos** [comment]
Another case here: 6421133-zd-woothemes

**Comment by DavidSaladu** [comment]
Another one here 6536677-zd-woothemes

**Comment by Robertght** [comment]
Another one in 6527930-zen

**Comment by sophiegyo** [comment]
Another case in https://wordpress.com/forums/topic/wix-import-2/

**Comment by filipanoscampos** [comment]
Another case, here: 6571272-zd-woothemes

**Comment by supernovia** [comment]
This one is producing a 403 error instead:
https://wordpress.com/forums/topic/help-how-to-migrate-websites-on-the-wix-platform-to-wordpress/

**Comment by CodeAllNightNDay** [comment]
403 import issue happening here 7393251-zen

**Comment by sajmes** [comment]
403 import issue here: 7528354-zd-a8c

**Comment by zachspears** [comment]
Additional in for the Forums: https://wordpress.com/forums/topic/wix-import-5/?view=all

**Comment by zachspears** [comment]
Additional 403 issue here: https://a8c.zendesk.com/agent/tickets/7675256

**Comment by zdenys** [comment]
7675992-zd-a8c

**Comment by zachspears** [comment]
Additional 403 issue here: 7675256-zd-a8c

**Comment by zemirajput** [comment]
Another one here:7805223-zd-a8c

**Comment by mgozdis** [comment]
8155297-zd-a8c - `It looks like we're unable to reach that website address at this time.`

**Comment by sanjeev00733** [comment]
Another report in 8224627-zd-a8c

**Comment by candy02058912** [comment]
Hi @Automattic/caribou, checking in on the status/priority of the issue? Can someone from the team own fixing this?

As part of pb5gDS-3YO-p2, we’re trying to make sure every blocker/high priority issue on  [The One Board](https://github.com/orgs/Automattic/projects/391/views/12) has an owner and clear next steps.

If prioritization isn't possible, please close the issue with the label "Won't Fix" and comment with the reason for future reference.

Thanks 🙏

**Comment by obenland** [comment]
This is definitely something that needs fixing, unfortunately it's not something that's straight forward. pcmemI-36d-p2 has more details.

To me, the next step would be: Prioritize work on this.

**Comment by candy02058912** [comment]
Thanks for sharing the P2 post which is helpful context.

> To me, the next step would be: Prioritize work on this.

To address prioritization, I noticed the newly created issue https://github.com/Automattic/dotcom-forge/issues/7841
but it's currently in the opportunity inbox.

@zaerl, do you have an idea of when the issue might be worked on, or know who I could ping to get a better idea? 🙏 

I'm marking this issue as a need product decision as it's not a trivial fix, and the new issues is in the opportunity inbox cc: @allilevine @ianstewart

**Comment by zaerl** [comment]
> @zaerl, do you have an idea of when the issue might be worked on, or know who I could ping to get a better idea? 🙏
It is under our radar, but I need to find out when my proposal for a new project will be considered. I haven't received additional information yet.

**Comment by alshakero** [comment]
Closing as fixed in pMz3w-kvS-p2#comment-120869
